packages/infra/config/env-config.ts

```diff
@@ -82,6 +82,9 @@ export type EnvConfig = {
     CW_GATEWAY_AUTHORIZATION_CLIENT_ID: string;
     CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET: string;
   };
+  ihe: {
+    CERT_ARN: string;
+  };
```

Comment:
Great that you created a new group! While we're here, can we move `iheSubdomain` here, please?